dcrec/secp256k1/field.go

```diff
@@ -1156,7 +1156,7 @@ func (f *FieldVal) SquareRootVal(val *FieldVal) bool {
 	// 10111111 11111111 11111111 00001100
 	//
 	// Notice that can be broken up into three windows of consecutive 1s (in
-	// order of least to most signifcant) as:
+	// order of least to most significant) as:
```

Comment:
Copyright year could be updated in this file as well.

Reply:
Good call.  I'm pretty sure I did it in a later commit, but it should be in this one.  Done.